lib/gracenote/client.rb

```diff
@@ -1,20 +1,22 @@
 require "gracenote/http"
 require "gracenote/query_builder"
 require "gracenote/errors"
-require "gracenote/search"
+require "gracenote/request/register"
+require "gracenote/request/search"
+require "gracenote/request/fetch"
 require "gracenote/response"
-require "gracenote/auth/registration"
 
 module Gracenote
   class Client
-    attr_reader :client_id, :user_id, :lang
+    attr_reader :client_id, :user_id, :lang, :country
 
     include Http
     include QueryBuilder
-    include Auth::Registration
-    include Search
+    include Request::Register
+    include Request::Search
+    include Request::Fetch
 
-    def initialize(client_id:, user_id: nil, lang: nil)
+    def initialize(client_id:, user_id: nil, lang: nil, country: nil)
```

Comment:
__[rubocop]__
* unexpected token tCOMMA
(Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
* cannot assign to a keyword
(Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
* cannot assign to a keyword
(Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)